{
  
    
        "post0": {
            "title": "Papers correlations EMAS 2019",
            "content": "EMAS2019_paper_5.pdf : From Goals to Organisations: automated organisation generator for MAS? Cleber Jo EMAS2019_paper_8.pdf : Jacamo-web is on the fly: an interactive Multi-Agent System IDE? Cleber Jorge Am EMAS2019_paper_18.pdf : SAT for Epistemic Logic using Belief Bases Fabián Romero1 and Emiliano Lorini1 EMAS2019_paper_21.pdf : JS-son - A Minimal JavaScript BDI Agent Library Timotheus Kampik and Juan Carlos EMAS2019_paper_22.pdf : An Architecture for Integrating BDI Agents with a Simulation Environment Alan Da EMAS2019_paper_23.pdf : Using MATSim as a Component in Dynamic Agent-Based Micro-Simulations Dhirendra S EMAS2019_paper_24.pdf : Plan Library Reconfigurability in BDI Agents? Rafael C. Cardoso, Louise A. Denni EMAS2019_paper_25.pdf : Incorporating social practices in BDI agent systems Stephen Cranefield1 and Fran EMAS2019_paper_26.pdf : Accountability and Agents for Engineering Business Processes Matteo Baldoni1, Cr EMAS2019_paper_27.pdf : The “Why did you do that?” Button: Answering Why-questions for end users of Robo EMAS2019_paper_28.pdf : Concurrency and Asynchrony in Protocol Languages Amit K. Chopra1, Samuel H. Chri EMAS2019_paper_29.pdf : On Enactability of Agent Interaction Protocols: Towards a Unified Approach Angel EMAS2019_paper_30.pdf : Who’s that? - Modelling Social Situations for Behaviour Support Agents? Ilir Kol EMAS2019_paper_31.pdf : What Does It Take to Create Social Awareness for Support Agents?? Ilir Kola1, Ca EMAS2019_paper_32.pdf : An Introduction to Engineering Multiagent Industrial Symbiosis Systems: Potentia EMAS2019_paper_33.pdf : From Programming Agents to Educating Agents – A Jason-based Framework for Integr EMAS2019_paper_34.pdf : Hercule: Reasoning about Norms over Unstructured Events Samuel H. Christie V1, A EMAS2019_paper_35.pdf : Agents are More Complex than Other Software: An Empirical Investigation Alon T. . How it works? . It downloads the accepted papers available in EMAS 2019 page. Each paper in PDF is converted to a plain text using Apache Tika. Then using Google Universal Sentence Encoder they are vectorized. These vectors are compared creating correlations. The correlations vary from 0 to 100% of similarity. Those values are presented in an Altair correlation matrix. . What else it can do? . I use it to find correlations across many papers and books I use in my researches. Since I use Mendeley, all of them are in a plain folder. The project called text-correlation retrieves all documents from a local folder creating a correlation matrix n x n in a .csv file. It is better for larger number of documents and suitable to open in a spreadsheet processor. . Credits . Developed by Cleber Jorge Amaral, acknowledging it is highly inspired by a work presented by Aladdin Shamoug .",
            "url": "https://cleberjamaral.github.io/fastpages/comparison/altair/jupyter/2020/05/30/text-correlations.html",
            "relUrl": "/comparison/altair/jupyter/2020/05/30/text-correlations.html",
            "date": " • May 30, 2020"
        }
        
    
  
    
        ,"post1": {
            "title": "Achate a curva [in Portuguese]",
            "content": "An Exception was encountered at &#39;In [19]&#39;. . Última atualização em 10/10/2020Total de casos confirmados: 5055888Total de recuperados: 4502854, 89% dos casosTotal de mortes: 149639, 3% dos casosTotal de ativos: 403395, 8% dos casos MaxRowsError Traceback (most recent call last) /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in to_dict(self, *args, **kwargs) 371 372 try: --&gt; 373 dct = super(TopLevelMixin, copy).to_dict(*args, **kwargs) 374 except jsonschema.ValidationError: 375 dct = None /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in to_dict(self, validate, ignore, context) 326 {k: v for k, v in self._kwds.items() if k not in ignore}, 327 validate=sub_validate, --&gt; 328 context=context, 329 ) 330 else: /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in _todict(obj, validate, context) 60 return { 61 k: _todict(v, validate, context) &gt; 62 for k, v in obj.items() 63 if v is not Undefined 64 } /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in &lt;dictcomp&gt;(.0) 61 k: _todict(v, validate, context) 62 for k, v in obj.items() &gt; 63 if v is not Undefined 64 } 65 elif hasattr(obj, &#34;to_dict&#34;): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in _todict(obj, validate, context) 56 return obj.to_dict(validate=validate, context=context) 57 elif isinstance(obj, (list, tuple, np.ndarray)): &gt; 58 return [_todict(v, validate, context) for v in obj] 59 elif isinstance(obj, dict): 60 return { /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in &lt;listcomp&gt;(.0) 56 return obj.to_dict(validate=validate, context=context) 57 elif isinstance(obj, (list, tuple, np.ndarray)): &gt; 58 return [_todict(v, validate, context) for v in obj] 59 elif isinstance(obj, dict): 60 return { /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in _todict(obj, validate, context) 54 &#34;&#34;&#34;Convert an object to a dict representation.&#34;&#34;&#34; 55 if isinstance(obj, SchemaBase): &gt; 56 return obj.to_dict(validate=validate, context=context) 57 elif isinstance(obj, (list, tuple, np.ndarray)): 58 return [_todict(v, validate, context) for v in obj] /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in to_dict(self, *args, **kwargs) 361 copy = self.copy(deep=False) 362 original_data = getattr(copy, &#34;data&#34;, Undefined) --&gt; 363 copy.data = _prepare_data(original_data, context) 364 365 if original_data is not Undefined: /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in _prepare_data(data, context) 82 # convert dataframes or objects with __geo_interface__ to dict 83 if isinstance(data, pd.DataFrame) or hasattr(data, &#34;__geo_interface__&#34;): &gt; 84 data = _pipe(data, data_transformers.get()) 85 86 # convert string input to a URLData /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/data.py in default_data_transformer(data, max_rows) 17 @curried.curry 18 def default_data_transformer(data, max_rows=5000): &gt; 19 return curried.pipe(data, limit_rows(max_rows=max_rows), to_values) 20 21 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/data.py in limit_rows(data, max_rows) 82 &#34;than the maximum allowed ({}). &#34; 83 &#34;For information on how to plot larger datasets &#34; &gt; 84 &#34;in Altair, see the documentation&#34;.format(max_rows) 85 ) 86 return data MaxRowsError: The number of rows in your dataset is greater than the maximum allowed (5000). For information on how to plot larger datasets in Altair, see the documentation . alt.LayerChart(...) . Estado Data Casos confirmados Ativos 1% dos ativos 5% dos ativos 12% dos ativos UTIs 35% das UTIs . 0 AC | 10/13/20 | 29272 | 2336 | 23 | 117 | 280 | 58 | 20 | . 2 AM | 10/13/20 | 147864 | 11798 | 118 | 590 | 1416 | 450 | 157 | . 3 AP | 10/13/20 | 49593 | 3957 | 40 | 198 | 475 | 78 | 27 | . 4 BA | 10/13/20 | 327327 | 26116 | 261 | 1306 | 3134 | 1573 | 550 | . 5 CE | 10/13/20 | 260642 | 20796 | 208 | 1040 | 2496 | 1002 | 350 | . 6 DF | 10/13/20 | 201230 | 16056 | 161 | 803 | 1927 | 997 | 348 | . 7 ES | 10/13/20 | 140701 | 11226 | 112 | 561 | 1347 | 804 | 281 | . 8 GO | 10/13/20 | 231182 | 18445 | 184 | 922 | 2213 | 1200 | 420 | . 9 MA | 10/13/20 | 179170 | 14295 | 143 | 715 | 1715 | 682 | 238 | . 10 MG | 10/13/20 | 324626 | 25901 | 259 | 1295 | 3108 | 3151 | 1102 | . 11 MS | 10/13/20 | 75076 | 5990 | 60 | 300 | 719 | 432 | 151 | . 12 MT | 10/13/20 | 132949 | 10608 | 106 | 530 | 1273 | 591 | 206 | . 13 PA | 10/13/20 | 238835 | 19056 | 191 | 953 | 2287 | 661 | 231 | . 14 PB | 10/13/20 | 126073 | 10059 | 101 | 503 | 1207 | 536 | 187 | . 15 PE | 10/13/20 | 153299 | 12231 | 122 | 612 | 1468 | 1647 | 576 | . 16 PI | 10/13/20 | 103461 | 8255 | 83 | 413 | 991 | 293 | 102 | . 17 PR | 10/13/20 | 194689 | 15534 | 155 | 777 | 1864 | 2444 | 855 | . 18 RJ | 10/13/20 | 284053 | 22664 | 227 | 1133 | 2720 | 4008 | 1402 | . 19 RN | 10/13/20 | 73423 | 5858 | 59 | 293 | 703 | 526 | 184 | . 20 RO | 10/13/20 | 68126 | 5436 | 54 | 272 | 652 | 285 | 99 | . 21 RR | 10/13/20 | 52701 | 4205 | 42 | 210 | 505 | 25 | 8 | . 22 RS | 10/13/20 | 214970 | 17152 | 172 | 858 | 2058 | 1660 | 581 | . 23 SC | 10/13/20 | 228403 | 18224 | 182 | 911 | 2187 | 1064 | 372 | . 24 SE | 10/13/20 | 80234 | 6402 | 64 | 320 | 768 | 295 | 103 | . 25 SP | 10/13/20 | 1039029 | 82901 | 829 | 4145 | 9948 | 9269 | 3244 | . 26 TO | 10/13/20 | 71269 | 5686 | 57 | 284 | 682 | 161 | 56 | . Premissas : - O percentual de casos ativos do país está senso utilizado para estimar o percentual em cada estado. - O Brasil tem 34318 leitos de UTI adulto SUS e não SUS (fonte: painel de insumos e leitos) - 35% de UTIs disponíveis para pacientes com covid19 (fonte: artigo oglobo) - Estima-se que cerca de 5% dos infectados por covid-19 venham a precisar de tratamento intensivo. No pior caso registrado, na Itália, cerca de 12% precisaram de UTI e no melhor caso, China, cerca de 1% dos infectados precisaram de UTI (fonte: artigo the lancet) - As estimativas são feitas para o número de infectados. Nesta simplificação, estas estimativas estão sendo extrapolada para o número de ativos. . Última atualização em 13/10/2020 Based on the work of Alonso Silva Allende and Joao B. Duarte, adapted by Cleber Jorge Amaral. . Dados: . covid19: CSSEGISandData | quantidade de leitos: painel de insumos e leitos, elpais, elpais2, bcc e ministério da saúde | leitos disponíveis: artigo oglobo | pacientes que precisam de UTI: artigo the lancet | . Execution using papermill encountered an exception here and stopped: .",
            "url": "https://cleberjamaral.github.io/fastpages/covid-19/brasil/comparativo/altair/jupyter/2020/05/11/brazil-flatten-the-curve_per_state.html",
            "relUrl": "/covid-19/brasil/comparativo/altair/jupyter/2020/05/11/brazil-flatten-the-curve_per_state.html",
            "date": " • May 11, 2020"
        }
        
    
  
    
        ,"post2": {
            "title": "Comparativo de novos casos por dia por estado [in Portuguese]",
            "content": "An Exception was encountered at &#39;In [8]&#39;. . NOTA . Os dados utilizados nestas análises são inerentemente incertos. Não há garantia de que os dados e as análises estejam devidamente atualizados e corretos. Antes de tomar conclusões é fundamental verificar dados e conclusões com outras fontes. . MaxRowsError Traceback (most recent call last) /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in to_dict(self, *args, **kwargs) 361 copy = self.copy(deep=False) 362 original_data = getattr(copy, &#34;data&#34;, Undefined) --&gt; 363 copy.data = _prepare_data(original_data, context) 364 365 if original_data is not Undefined: /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in _prepare_data(data, context) 82 # convert dataframes or objects with __geo_interface__ to dict 83 if isinstance(data, pd.DataFrame) or hasattr(data, &#34;__geo_interface__&#34;): &gt; 84 data = _pipe(data, data_transformers.get()) 85 86 # convert string input to a URLData /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/data.py in default_data_transformer(data, max_rows) 17 @curried.curry 18 def default_data_transformer(data, max_rows=5000): &gt; 19 return curried.pipe(data, limit_rows(max_rows=max_rows), to_values) 20 21 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/data.py in limit_rows(data, max_rows) 82 &#34;than the maximum allowed ({}). &#34; 83 &#34;For information on how to plot larger datasets &#34; &gt; 84 &#34;in Altair, see the documentation&#34;.format(max_rows) 85 ) 86 return data MaxRowsError: The number of rows in your dataset is greater than the maximum allowed (5000). For information on how to plot larger datasets in Altair, see the documentation . alt.HConcatChart(...) . MaxRowsError Traceback (most recent call last) /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in to_dict(self, *args, **kwargs) 361 copy = self.copy(deep=False) 362 original_data = getattr(copy, &#34;data&#34;, Undefined) --&gt; 363 copy.data = _prepare_data(original_data, context) 364 365 if original_data is not Undefined: /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in _prepare_data(data, context) 82 # convert dataframes or objects with __geo_interface__ to dict 83 if isinstance(data, pd.DataFrame) or hasattr(data, &#34;__geo_interface__&#34;): &gt; 84 data = _pipe(data, data_transformers.get()) 85 86 # convert string input to a URLData /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/data.py in default_data_transformer(data, max_rows) 17 @curried.curry 18 def default_data_transformer(data, max_rows=5000): &gt; 19 return curried.pipe(data, limit_rows(max_rows=max_rows), to_values) 20 21 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/data.py in limit_rows(data, max_rows) 82 &#34;than the maximum allowed ({}). &#34; 83 &#34;For information on how to plot larger datasets &#34; &gt; 84 &#34;in Altair, see the documentation&#34;.format(max_rows) 85 ) 86 return data MaxRowsError: The number of rows in your dataset is greater than the maximum allowed (5000). For information on how to plot larger datasets in Altair, see the documentation . alt.HConcatChart(...) . Última atualização em 13/10/2020 Based on works developed by Alonso Silva Allende, adapted by Cleber Jorge Amaral. Data source: brasil.io . Execution using papermill encountered an exception here and stopped: .",
            "url": "https://cleberjamaral.github.io/fastpages/covid-19/brasil/comparativo/altair/jupyter/2020/05/08/brazil-states-linear-cases-and-deaths-newcases.html",
            "relUrl": "/covid-19/brasil/comparativo/altair/jupyter/2020/05/08/brazil-states-linear-cases-and-deaths-newcases.html",
            "date": " • May 8, 2020"
        }
        
    
  
    
        ,"post3": {
            "title": "Comparativo casos acumulados por estado [in Portuguese]",
            "content": "An Exception was encountered at &#39;In [9]&#39;. . NOTA . Os dados utilizados nestas análises são inerentemente incertos. Não há garantia de que os dados e as análises estejam devidamente atualizados e corretos. Antes de tomar conclusões é fundamental verificar dados e conclusões com outras fontes. . MaxRowsError Traceback (most recent call last) /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in to_dict(self, *args, **kwargs) 361 copy = self.copy(deep=False) 362 original_data = getattr(copy, &#34;data&#34;, Undefined) --&gt; 363 copy.data = _prepare_data(original_data, context) 364 365 if original_data is not Undefined: /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in _prepare_data(data, context) 82 # convert dataframes or objects with __geo_interface__ to dict 83 if isinstance(data, pd.DataFrame) or hasattr(data, &#34;__geo_interface__&#34;): &gt; 84 data = _pipe(data, data_transformers.get()) 85 86 # convert string input to a URLData /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/data.py in default_data_transformer(data, max_rows) 17 @curried.curry 18 def default_data_transformer(data, max_rows=5000): &gt; 19 return curried.pipe(data, limit_rows(max_rows=max_rows), to_values) 20 21 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/data.py in limit_rows(data, max_rows) 82 &#34;than the maximum allowed ({}). &#34; 83 &#34;For information on how to plot larger datasets &#34; &gt; 84 &#34;in Altair, see the documentation&#34;.format(max_rows) 85 ) 86 return data MaxRowsError: The number of rows in your dataset is greater than the maximum allowed (5000). For information on how to plot larger datasets in Altair, see the documentation . alt.HConcatChart(...) . MaxRowsError Traceback (most recent call last) /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in to_dict(self, *args, **kwargs) 361 copy = self.copy(deep=False) 362 original_data = getattr(copy, &#34;data&#34;, Undefined) --&gt; 363 copy.data = _prepare_data(original_data, context) 364 365 if original_data is not Undefined: /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in _prepare_data(data, context) 82 # convert dataframes or objects with __geo_interface__ to dict 83 if isinstance(data, pd.DataFrame) or hasattr(data, &#34;__geo_interface__&#34;): &gt; 84 data = _pipe(data, data_transformers.get()) 85 86 # convert string input to a URLData /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/data.py in default_data_transformer(data, max_rows) 17 @curried.curry 18 def default_data_transformer(data, max_rows=5000): &gt; 19 return curried.pipe(data, limit_rows(max_rows=max_rows), to_values) 20 21 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/data.py in limit_rows(data, max_rows) 82 &#34;than the maximum allowed ({}). &#34; 83 &#34;For information on how to plot larger datasets &#34; &gt; 84 &#34;in Altair, see the documentation&#34;.format(max_rows) 85 ) 86 return data MaxRowsError: The number of rows in your dataset is greater than the maximum allowed (5000). For information on how to plot larger datasets in Altair, see the documentation . alt.HConcatChart(...) . Última atualização em 13/10/2020 Based on works developed by Alonso Silva Allende, adapted by Cleber Jorge Amaral. Data source: brasil.io . Execution using papermill encountered an exception here and stopped: .",
            "url": "https://cleberjamaral.github.io/fastpages/covid-19/brasil/comparativo/altair/jupyter/2020/05/08/brazil-states-linear-case-and-deaths-accumulated.html",
            "relUrl": "/covid-19/brasil/comparativo/altair/jupyter/2020/05/08/brazil-states-linear-case-and-deaths-accumulated.html",
            "date": " • May 8, 2020"
        }
        
    
  
    
        ,"post4": {
            "title": "Comparativo trajetória por estado [in Portuguese]",
            "content": "An Exception was encountered at &#39;In [21]&#39;. . NOTA . Os dados utilizados nestas análises são inerentemente incertos. Não há garantia de que os dados e as análises estejam devidamente atualizados e corretos. Antes de tomar conclusões é fundamental verificar dados e conclusões com outras fontes. . MaxRowsError Traceback (most recent call last) /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in to_dict(self, *args, **kwargs) 371 372 try: --&gt; 373 dct = super(TopLevelMixin, copy).to_dict(*args, **kwargs) 374 except jsonschema.ValidationError: 375 dct = None /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in to_dict(self, validate, ignore, context) 326 {k: v for k, v in self._kwds.items() if k not in ignore}, 327 validate=sub_validate, --&gt; 328 context=context, 329 ) 330 else: /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in _todict(obj, validate, context) 60 return { 61 k: _todict(v, validate, context) &gt; 62 for k, v in obj.items() 63 if v is not Undefined 64 } /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in &lt;dictcomp&gt;(.0) 61 k: _todict(v, validate, context) 62 for k, v in obj.items() &gt; 63 if v is not Undefined 64 } 65 elif hasattr(obj, &#34;to_dict&#34;): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in _todict(obj, validate, context) 56 return obj.to_dict(validate=validate, context=context) 57 elif isinstance(obj, (list, tuple, np.ndarray)): &gt; 58 return [_todict(v, validate, context) for v in obj] 59 elif isinstance(obj, dict): 60 return { /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in &lt;listcomp&gt;(.0) 56 return obj.to_dict(validate=validate, context=context) 57 elif isinstance(obj, (list, tuple, np.ndarray)): &gt; 58 return [_todict(v, validate, context) for v in obj] 59 elif isinstance(obj, dict): 60 return { /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in _todict(obj, validate, context) 54 &#34;&#34;&#34;Convert an object to a dict representation.&#34;&#34;&#34; 55 if isinstance(obj, SchemaBase): &gt; 56 return obj.to_dict(validate=validate, context=context) 57 elif isinstance(obj, (list, tuple, np.ndarray)): 58 return [_todict(v, validate, context) for v in obj] /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in to_dict(self, *args, **kwargs) 361 copy = self.copy(deep=False) 362 original_data = getattr(copy, &#34;data&#34;, Undefined) --&gt; 363 copy.data = _prepare_data(original_data, context) 364 365 if original_data is not Undefined: /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in _prepare_data(data, context) 82 # convert dataframes or objects with __geo_interface__ to dict 83 if isinstance(data, pd.DataFrame) or hasattr(data, &#34;__geo_interface__&#34;): &gt; 84 data = _pipe(data, data_transformers.get()) 85 86 # convert string input to a URLData /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/data.py in default_data_transformer(data, max_rows) 17 @curried.curry 18 def default_data_transformer(data, max_rows=5000): &gt; 19 return curried.pipe(data, limit_rows(max_rows=max_rows), to_values) 20 21 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/data.py in limit_rows(data, max_rows) 82 &#34;than the maximum allowed ({}). &#34; 83 &#34;For information on how to plot larger datasets &#34; &gt; 84 &#34;in Altair, see the documentation&#34;.format(max_rows) 85 ) 86 return data MaxRowsError: The number of rows in your dataset is greater than the maximum allowed (5000). For information on how to plot larger datasets in Altair, see the documentation . alt.LayerChart(...) . MaxRowsError Traceback (most recent call last) /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in to_dict(self, *args, **kwargs) 371 372 try: --&gt; 373 dct = super(TopLevelMixin, copy).to_dict(*args, **kwargs) 374 except jsonschema.ValidationError: 375 dct = None /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in to_dict(self, validate, ignore, context) 326 {k: v for k, v in self._kwds.items() if k not in ignore}, 327 validate=sub_validate, --&gt; 328 context=context, 329 ) 330 else: /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in _todict(obj, validate, context) 60 return { 61 k: _todict(v, validate, context) &gt; 62 for k, v in obj.items() 63 if v is not Undefined 64 } /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in &lt;dictcomp&gt;(.0) 61 k: _todict(v, validate, context) 62 for k, v in obj.items() &gt; 63 if v is not Undefined 64 } 65 elif hasattr(obj, &#34;to_dict&#34;): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in _todict(obj, validate, context) 56 return obj.to_dict(validate=validate, context=context) 57 elif isinstance(obj, (list, tuple, np.ndarray)): &gt; 58 return [_todict(v, validate, context) for v in obj] 59 elif isinstance(obj, dict): 60 return { /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in &lt;listcomp&gt;(.0) 56 return obj.to_dict(validate=validate, context=context) 57 elif isinstance(obj, (list, tuple, np.ndarray)): &gt; 58 return [_todict(v, validate, context) for v in obj] 59 elif isinstance(obj, dict): 60 return { /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/schemapi.py in _todict(obj, validate, context) 54 &#34;&#34;&#34;Convert an object to a dict representation.&#34;&#34;&#34; 55 if isinstance(obj, SchemaBase): &gt; 56 return obj.to_dict(validate=validate, context=context) 57 elif isinstance(obj, (list, tuple, np.ndarray)): 58 return [_todict(v, validate, context) for v in obj] /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in to_dict(self, *args, **kwargs) 361 copy = self.copy(deep=False) 362 original_data = getattr(copy, &#34;data&#34;, Undefined) --&gt; 363 copy.data = _prepare_data(original_data, context) 364 365 if original_data is not Undefined: /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/v4/api.py in _prepare_data(data, context) 82 # convert dataframes or objects with __geo_interface__ to dict 83 if isinstance(data, pd.DataFrame) or hasattr(data, &#34;__geo_interface__&#34;): &gt; 84 data = _pipe(data, data_transformers.get()) 85 86 # convert string input to a URLData /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/vegalite/data.py in default_data_transformer(data, max_rows) 17 @curried.curry 18 def default_data_transformer(data, max_rows=5000): &gt; 19 return curried.pipe(data, limit_rows(max_rows=max_rows), to_values) 20 21 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in pipe(data, *funcs) 625 &#34;&#34;&#34; 626 for func in funcs: --&gt; 627 data = func(data) 628 return data 629 /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/toolz/functoolz.py in __call__(self, *args, **kwargs) 301 def __call__(self, *args, **kwargs): 302 try: --&gt; 303 return self._partial(*args, **kwargs) 304 except TypeError as exc: 305 if self._should_curry(args, kwargs, exc): /opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/altair/utils/data.py in limit_rows(data, max_rows) 82 &#34;than the maximum allowed ({}). &#34; 83 &#34;For information on how to plot larger datasets &#34; &gt; 84 &#34;in Altair, see the documentation&#34;.format(max_rows) 85 ) 86 return data MaxRowsError: The number of rows in your dataset is greater than the maximum allowed (5000). For information on how to plot larger datasets in Altair, see the documentation . alt.LayerChart(...) . Última atualização em 13/10/2020 Based on the notebook developed by Pratap Vardhan, adapted by Cleber Jorge Amaral. Data source: brasil.io . Execution using papermill encountered an exception here and stopped: .",
            "url": "https://cleberjamaral.github.io/fastpages/covid-19/brasil/comparativo/altair/jupyter/2020/05/05/brazil-states-trajectories-and-deaths.html",
            "relUrl": "/covid-19/brasil/comparativo/altair/jupyter/2020/05/05/brazil-states-trajectories-and-deaths.html",
            "date": " • May 5, 2020"
        }
        
    
  

  
  

  
      ,"page1": {
          "title": "About Me",
          "content": "Please, go to my personal website. .",
          "url": "https://cleberjamaral.github.io/fastpages/about/",
          "relUrl": "/about/",
          "date": ""
      }
      
  

  

  
  

  

  
  

  

  
  

  
  

  
  

  
      ,"page10": {
          "title": "",
          "content": "Sitemap: {{ “sitemap.xml” | absolute_url }} | .",
          "url": "https://cleberjamaral.github.io/fastpages/robots.txt",
          "relUrl": "/robots.txt",
          "date": ""
      }
      
  

}